XmlFormatterOsIndependent/Commands/UpdateApplicationCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using PluginFramework.Interfaces.PluginTypes;
+using System;
+using System.Windows.Input;
+using XmlFormatterOsIndependent.DataSets;
+
+namespace XmlFormatterOsIndependent.Commands
+{
+    /// <summary>
+    /// This class will check for new updates and try to update the application
+    /// </summary>
+    internal class UpdateApplicationCommand : ICommand
+    {
+        /// <inheritdoc/>
+        public event EventHandler CanExecuteChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'CanExecuteChanged' or invoke it.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)